backend/endpoints/telegrambot.py

```diff
@@ -37,6 +37,20 @@ def send_request(beneficiary):
             pass
 
 
+def send_assign(beneficiary):
+    """Sends a POST request to telegrambot api to assign a request to a volunteer
+    :param beneficiary: Beneficiary data for whom will be performed a request
+    """
+    payload = {
+        'request_id': str(beneficiary['id']),
+        'volunteer': str(beneficiary['volunteer'])
```

Comment:
There should also be a `time` component, see the readme of the bot itself:

```
{
    "request_id": "fe91e4b6-e902-4d03-8500-d058673cb9bd",
    "volunteer": chat_id,
    "time": "20:45"
}
```

Reply:
Also, keep in mind that `volunteer` in this case is their Telegram `chat_id`.

Reply:
Done